### PR TITLE
aligned meshery operator with the heading

### LINKED ISSUE #7279: [Docs] Update alignment of icon in Meshery Operator page
#### Current State

Screenshot from 2023-03-05 13-59-22

#### Desired State
Screenshot from 2023-03-05 14-00-25

(Icon to be visible beside heading)

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Ambareen09**
**Notes for Reviewers**

This PR fixes #7279 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
Updated the size of the icon to align it wth the heading

- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 93aaea40e63997ced84760a9fceff02b65ebb01f at: https://6404cb04c15eee320a63d772--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c51a22d59c58f78019bf4893029a10e70783ad29 at: https://6404d6729f540d44880ef772--meshery-docs-preview.netlify.app

**Review by gurpreet-legend — APPROVED** [review]

File: docs/pages/concepts/architecture/operator.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ display-title: "false"
 list: include
 ---
 
-# Meshery Operator <img src="{{ site.baseurl }}/assets/img/architecture/B203EFA85E89491B.png"/>
+# Meshery Operator <img src="{{ site.baseurl }}/assets/img/architecture/B203EFA85E89491B.png" width="20px"/>
```
Comment (leecalcote) [inline docs/pages/concepts/architecture/operator.md:13]: Thanks for contributing, @Ambareen09 👍 

Please use dynamic styling (e.g flex with justify-content) as opposed to static sizing that suffers fragility and will inevitably not be appropriately positioned depending upon the scenario.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 5d0ac6a693c2af91f06e01e25a581b56aa1313c8 at: https://64058b670dfed03091d8efb8--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 640ebc15d23b1f3c1d80a523880a2b38394c2548 at: https://64058cb4ac95be43d4619bac--meshery-docs-preview.netlify.app

**Review by gurpreet-legend — COMMENTED** [review]
Below "500px", the icon becomes too small, need to have some media query for this

image

**Comment by leecalcote** [comment]
min-width, max-width, flex, ... helpful for dynamic css.

**Comment by l5io** [comment]
🚀 Preview for commit 9cadbdca145c3332c17316ab5ab850666aeed390 at: https://64070c0eda40386753daf338--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 81565eed543e7b4363cd4827f2087116ad467b5c at: https://64070de8ccc0566bccf6fda7--meshery-docs-preview.netlify.app

**Comment by Ambareen09** [comment]
I think it's responsive now, can you please have a look now?

**Comment by leecalcote** [comment]
@Ambareen09 Nope, not quite. I think the concept of static vs dynamic CSS alludes us here. Try again. Fourth time is a charm?

**Comment by l5io** [comment]
🚀 Preview for commit add2ec1fb5fcad776d52d50fa1ef27fb6cc73f11 at: https://640866b06eb2b2173e0dd3fc--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 230703b1d2325ef93f1274e7c14eaf230674bef0 at: https://6408688c005808167d6984bc--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 669b75fe2f279dbedbde4de2e464a6a01125f317 at: https://64086cb99903cc1771391310--meshery-docs-preview.netlify.app

**Comment by Ambareen09** [comment]
Hey, I believe CSS is not working in the .md file, can you please suggest anything which would make me resolve this issue? I am unable to use the style tags in the .md file as well (separately) and the media query isn't working with inline style. can you help me figure out how to work?

**Comment by Chadha93** [comment]
@Ambareen09  Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by Ambareen09** [comment]
Sure, could you please give me access to edit so that I can add my name? Or you can fix the meeting at your convenience anytime today.
My email is ambubushra@gmail.com

**Comment by Chadha93** [comment]
> Sure, could you please give me access to edit so that I can add my name? Or you can fix the meeting at your convenience anytime today @Chadha93 My email is [ambubushra@gmail.com](mailto:ambubushra@gmail.com)

@Ambareen09 Please fill this form to access - [layer5.io/newcomers](https://layer5.io/newcomers). You can add your items as suggestions.

**Comment by Ambareen09** [comment]
> > Sure, could you please give me access to edit so that I can add my name? Or you can fix the meeting at your convenience anytime today @Chadha93 My email is [ambubushra@gmail.com](mailto:ambubushra@gmail.com)
> 
> @Ambareen09 Please fill this form to access - [layer5.io/newcomers](https://layer5.io/newcomers). You can add your items as suggestions.

Added in list of topics as suggestions. When will be the call?

**Comment by l5io** [comment]
🚀 Preview for commit e47edcd885cb1eab8d4a1cfbe3b9f05071df91d4 at: https://6408a98bcd34d75c22d82e56--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@Ambareen09 kudos to you for not giving up. 

> > > Sure, could you please give me access to edit so that I can add my name? Or you can fix the meeting at your convenience anytime today @Chadha93 My email is [ambubushra@gmail.com](mailto:ambubushra@gmail.com)
> > 
> > 
> > @Ambareen09 Please fill this form to access - [layer5.io/newcomers](https://layer5.io/newcomers). You can add your items as suggestions.
> 
> Added in list of topics as suggestions. When will be the call?

@Ambareen09 regarding when the call will be: it's today (every Monday). Click the big yellow button on https://meet.layer5.io subscribe to the community calendar.

**Comment by leecalcote** [comment]
This CSS styling isn't what I'm referring to when asking for dynamic styling, but let's move forward anyway.

**Comment by Ambareen09** [comment]
Thank you for accepting my first PR!

**Pull request merged**